src/main/java/com/pirimid/cryptotrade/websocket/gemini/handler/GeminiSessionHandler.java

```diff
@@ -0,0 +1,92 @@
+package com.pirimid.cryptotrade.websocket.gemini.handler;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.pirimid.cryptotrade.DTO.OrderResDTO;
+import com.pirimid.cryptotrade.DTO.TradeDto;
+import com.pirimid.cryptotrade.helper.exchange.EXCHANGE;
+import com.pirimid.cryptotrade.model.Account;
+import com.pirimid.cryptotrade.service.OrderService;
+import com.pirimid.cryptotrade.util.GeminiUtil;
+import com.pirimid.cryptotrade.websocket.gemini.response.OrderResponse;
+import org.springframework.web.socket.CloseStatus;
+import org.springframework.web.socket.WebSocketHandler;
+import org.springframework.web.socket.WebSocketMessage;
+import org.springframework.web.socket.WebSocketSession;
+
+import java.util.List;
+
+public class GeminiSessionHandler implements WebSocketHandler {
+    private boolean isConnected=false;
+    private OrderService orderService;
+    private Account account;
+
+    public GeminiSessionHandler(Account account,OrderService orderService){
+        this.orderService = orderService;
+        this.account=account;
+    }
+    @Override
+    public void afterConnectionEstablished(WebSocketSession session) throws Exception {
+    }
+
+    @Override
+    public void handleMessage(WebSocketSession session, WebSocketMessage<?> message) throws Exception {
+        String payload = message.getPayload().toString();
```

Comment:
why we need `isConnected` in gemini not in coinbase ?

Reply:
because if the subscription_ack is not recieved then we can't proceed further. It's kept for checking subscription_ack

Reply:
I have added same in coinbase too.